# Array of the feasibility of the skills
Feasibility[] feasibilities